### PR TITLE
TSPS-307 Retrieve workspace info from Rawls when updating control workspace

### PR DESCRIPTION
**Pull request by mmorgantaylor**
### Description 
- when calling the admin endpoint to update a pipeline's control workspace, we now need to provide only the workspace project and name (and wdl method version); we call Rawls to fetch the storage container bucket and google project
- add WorkspaceGoogleProject (pulled from Rawls)
- change WorkspaceStorageContainerUrl to WorkspaceStorageContainerName (without gs:// prefix)
- change WorkspaceProject to WorkspaceBillingProject, to distinguish from WorkspaceGoogleProject

### Jira Ticket
https://broadworkbench.atlassian.net/browse/TSPS-307

### CODE REVIEW

File: service/src/main/java/bio/terra/pipelines/db/entities/Pipeline.java [hunk 1]
```diff
@@ -116,7 +121,8 @@ public String toString() {
         .add("workspaceId=" + workspaceId)
         .add("workspaceProject=" + workspaceProject)
         .add("workspaceName=" + workspaceName)
-        .add("workspaceStorageContainerUrl=" + workspaceStorageContainerUrl)
+        .add("workspaceStorageContainerUrl=" + workspaceStorageContainerName)
```
Comment (jsotobroad) [inline service/src/main/java/bio/terra/pipelines/db/entities/Pipeline.java:124]: `workspaceStorageContainerUrl` -> `workspaceStorageContainerName` in the string part

File: service/src/main/java/bio/terra/pipelines/db/entities/PipelineRun.java [hunk 2]
```diff
@@ -46,8 +46,11 @@ public class PipelineRun {
   @Column(name = "workspace_name")
   private String workspaceName;
 
-  @Column(name = "workspace_storage_container_url", nullable = false)
-  private String workspaceStorageContainerUrl;
+  @Column(name = "workspace_storage_container_name", nullable = false)
+  private String workspaceStorageContainerName;
+
+  @Column(name = "workspace_google_project", nullable = false)
```
Comment (jsotobroad) [inline service/src/main/java/bio/terra/pipelines/db/entities/PipelineRun.java:52]: whats the difference between `workspace_google_project` and `workspace_project`?

**Review by jsotobroad — COMMENTED** [review]
a clarifying question.  once #116 is merged this will need a few more updates as well correct?

File: service/src/main/resources/db/changelog.xml [hunk 3]
```diff
@@ -23,6 +23,7 @@
   <include file="changesets/20240806_add_workspace_project_and_workspace_name.yaml" relativeToChangelogFile="true"/>
   <include file="changesets/20240820_add_workspace_storage_container_url.yaml" relativeToChangelogFile="true"/>
   <include file="changesets/20240824_add_wdl_version_update_pipeline_version.yaml" relativeToChangelogFile="true"/>
+    <include file="changesets/20240830_add_workspace_google_project_rename_storage_container_field.yaml" relativeToChangelogFile="true"/>
```
Comment (jsotobroad) [inline service/src/main/resources/db/changelog.xml:26]: weird spacing

File: service/src/main/java/bio/terra/pipelines/db/entities/PipelineRun.java (hunk 2 above)
Comment (jsotobroad) [reply service/src/main/java/bio/terra/pipelines/db/entities/PipelineRun.java:52]: chatted with faces, think we can rename `workspace_project` to `workspace_billing_project` or `workspace_terra_project` to be more clear

File: service/src/main/java/bio/terra/pipelines/db/entities/Pipeline.java (hunk 1 above)
Comment (mmorgantaylor) [reply service/src/main/java/bio/terra/pipelines/db/entities/Pipeline.java:124]: nice catch, thanks!

File: service/src/main/java/bio/terra/pipelines/stairway/imputation/RunImputationGcpJobFlight.java [hunk 4]
```diff
@@ -39,7 +39,7 @@ public RunImputationGcpJobFlight(FlightMap inputParameters, Object beanBag) {
         RunImputationJobFlightMapKeys.PIPELINE_INPUT_DEFINITIONS,
         RunImputationJobFlightMapKeys.PIPELINE_OUTPUT_DEFINITIONS,
         RunImputationJobFlightMapKeys.USER_PROVIDED_PIPELINE_INPUTS,
-        RunImputationJobFlightMapKeys.CONTROL_WORKSPACE_PROJECT,
+        RunImputationJobFlightMapKeys.CONTROL_WORKSPACE_BILLING_PROJECT,
         RunImputationJobFlightMapKeys.CONTROL_WORKSPACE_NAME,
         RunImputationJobFlightMapKeys.CONTROL_WORKSPACE_STORAGE_CONTAINER_URL,
```
Comment (jsotobroad) [inline service/src/main/java/bio/terra/pipelines/stairway/imputation/RunImputationGcpJobFlight.java:44]: should we update this here or in a future PR when we start wiring through the inputs?

**Review by jsotobroad — APPROVED** [review]
one comment about whether we should do some refactoring now or later but looks good otherwise!!

File: service/src/main/java/bio/terra/pipelines/stairway/imputation/RunImputationGcpJobFlight.java (hunk 4 above)
Comment (mmorgantaylor) [reply service/src/main/java/bio/terra/pipelines/stairway/imputation/RunImputationGcpJobFlight.java]: oh good catch, should do it here.

**Pull request merged**